src/Stratis.Bitcoin.Features.Consensus/Rules/ProvenHeaderRules/ProvenHeaderValidationConsensusRuleBase.cs

```diff
@@ -0,0 +1,58 @@
+﻿using NBitcoin;
+using Stratis.Bitcoin.Consensus.Rules;
+using Stratis.Bitcoin.Networks;
+using Stratis.Bitcoin.Utilities;
+
+namespace Stratis.Bitcoin.Features.Consensus.Rules.ProvenHeaderRules
+{
+    /// <summary>
+    /// Base rule to be used by all proven header validation rules. 
+    /// </summary>
+    /// <seealso cref="Stratis.Bitcoin.Consensus.Rules.HeaderValidationConsensusRule" />
+    public abstract class ProvenHeaderValidationConsensusRuleBase : HeaderValidationConsensusRule
```

Comment:
Ah ok fine `ProvenHeaderValidationConsensusRuleBase ` is part of `HeaderValidationConsensusRule`